.env.example

```diff
@@ -134,8 +134,8 @@ NEXT_PUBLIC_TEAM_IMPERSONATION=false
 # Close.com internal CRM
 CLOSECOM_API_KEY=
 
-# Sendgrid internal email sender
-SENDGRID_API_KEY=
+# Sendgrid internal sync service
+SENDGRID_SYNC_API_KEY=
```

Comment:
Differentiated the Sendgrid API Key from the one we use for Sending emails as they could not be the same.